services/postprocessing/pkg/service/service.go

```diff
@@ -73,3 +78,49 @@ func (pps *PostprocessingService) Run() error {
 	}
 	return nil
 }
+
+func getSteps(c config.Postprocessing) []events.Postprocessingstep {
+	// NOTE: improved version only allows configuring order of postprocessing steps
+	// But we aim for a system where postprocessing steps can be configured per space, ideally by the spaceadmin itself
+	// We need to iterate over configuring PP service when we see fit
+	var steps []events.Postprocessingstep
+	for _, s := range c.Steps {
+		steps = append(steps, events.Postprocessingstep(s))
+	}
+
+	if c.Virusscan {
+		if !contains(steps, events.PPStepAntivirus) {
+			steps = append(steps, events.PPStepAntivirus)
+			fmt.Printf("ATTENTION: POSTPROCESSING_VIRUSSCAN is deprecated. Use `POSTPROCESSING_STEPS=%v` in the future\n", join(steps))
```

Comment:
hm, I would expect this kind of env var parsing / config validation to live in `services/postprocessing/pkg/config/persier/parse.go`

Reply:
OK. Moved it to validate func